Mobile/ShopLens/Droid/Helpers/ImageRecognizer.cs

```diff
@@ -0,0 +1,169 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Android.Content;
+using Android.Gms.Vision;
+using Android.Gms.Vision.Texts;
+using Android.Graphics;
+using Android.Speech.Tts;
+using Camera;
+using ImageRecognitionMobile;
+using ImageRecognitionMobile.Classificators;
+using ShopLens.Droid.Helpers;
+using ShopLens.Droid.Notifications;
+using ShopLens.Droid.Source;
+using ShopLens.Extensions;
+using Unity;
+using PCLAppConfig;
+using Android.App;
+using ImageRecognitionMobile.OCR;
+
+namespace ShopLens.Droid
+{
+    class ImageRecognizer
+    {
+        TextToSpeech tts;
+        bool isTtsInitialized = false;
+        ShopLensTextToSpeech sltts;
+        TextRecognizer textRecognizer;
+        Context context;
+        Activity activity;
+        Bitmap bitmap;
+        Camera2Fragment owner;
+        private readonly string prefsName = ConfigurationManager.AppSettings["ShopCartPrefs"];
+
+        public ImageRecognizer(Context context, Activity activity)
+        {
+            this.activity = activity;
+            this.context = context;
+            textRecognizer = new TextRecognizer.Builder(context).Build();
+            sltts = new ShopLensTextToSpeech(context, OnTextToSpeechInit, OnTextToSpeechEndOfSpeech);
+            tts = new TextToSpeech(context, sltts);
```

Comment:
You don't need to create a tts object when sltts is doing all the talking for you.